src/components/Carousel/index.tsx

```diff
@@ -218,21 +218,20 @@ const navigationPositionStyle: { [key in CarouselNavigationPosition]: FlattenSim
 const StyledSwiper = styled(Swiper)<StyledSwiperProps>`
   &.swiper-container {
     ${props =>
-      props.lgSlidesSideOffset
-        ? `
-            padding-left: ${props.lgSlidesSideOffset}px;
-            padding-right: ${props.lgSlidesSideOffset}px;
-          `
-        : ''};
+      props.lgSlidesSideOffset &&
```

Comment:
number를 conditional statement에 그대로 사용할 경우, 0 도 falsy한 값으로 판단된다는 점을 유의하셔야 합니당

예기치 않은 문제를 방지하려면
```suggestion
      props.lgSlidesSideOffset !== undefined &&
```

이렇게 boolean expression 으로 바꿔주는 것이 안전

Reply:
아하아하 땡큐